lib/main.dart

```diff
@@ -20,15 +20,12 @@ $stack''';
 }
 
 Future<void> reportError(dynamic ex, StackTrace stack) async {
-  if (Prefs.devMode.get()) {
-    ShowFunctions.instance.showError(
-      errorDetails: buildErrorReport(ex, stack),
-    );
+  if (Prefs.isInitialized && Prefs.devMode.get()) {
```

Comment:
```suggestion
  if (kDebugMode || (Prefs.isInitialized && Prefs.devMode.get())) {
```
Should we maybe just always show the error toast if we were build in debug mode?

Reply:
It was a bit annoying in the past, because it could generate subsequent errors from the original one

Reply:
Yeah, I guess it's better to leave it as an explicit setting. 👍